modules/plugin-search.php

```diff
@@ -6,17 +6,19 @@
 	exit;
 }
 
-if (
-	is_admin() &&
-	Jetpack::is_active() &&
-	/** This filter is documented in _inc/lib/admin-pages/class.jetpack-react-page.php */
-	apply_filters( 'jetpack_show_promotions', true )
-) {
-	Jetpack_Plugin_Search::init();
-}
+if ( jetpack_is_psh_active() ) {
```

Comment:
Should we move this lower after the other checks? Will be a useless request if any of the other conditions are true.

Reply:
I added it only on the big condition. We can leave the line that initializes the rest api alone.